utils/utils.go

```diff
@@ -25,6 +25,8 @@ const (
 	MainnetChainID = "evmos_9001"
 	// TestnetChainID defines the Evmos EIP155 chain ID for testnet
 	TestnetChainID = "evmos_9000"
+	// TestingChainID defines the Evmos EIP155 chain ID for integration test
+	TestingChainID = "test_9000"
```

Comment:
In the existing integration tests we're not using this.
The comment is a little confusing. What is the main purpose of adding this new chain ID?
Also, if it is used only in tests, would be better to move it to a tests-related package, like testutils or some utils_test package

Reply:
This is only for testing, but the logic to handle it was added to the evm hook.